sdcm/nemesis.py

```diff
@@ -6660,3 +6711,21 @@ class EndOfQuotaNemesis(Nemesis):
 
     def disrupt(self):
         self.disrupt_end_of_quota_nemesis()
+
+
+class GrowShrinkZeroTokenNode(Nemesis):
+
+    disruptive = True
```

Comment:
I think we used this flag to exclude 2 parallel disruptive nemesis to avoid issues with CL=QUORUM, but here I don't consider this as an issue. Maybe these should not be marked as disruptive?

Reply:
Present new flag